src/main/java/com/conveyal/gtfs/error/DuplicateStopError.java

```diff
@@ -11,9 +11,9 @@
     private final String message;
     public final DuplicateStops duplicateStop;
 
-    public DuplicateStopError(String message, DuplicateStops duplicateStop) {
-        super("stop", 0, "stop_lat,stop_lon");
-        this.message = message;
+    public DuplicateStopError(DuplicateStops duplicateStop) {
+        super("stop", duplicateStop.stop1.sourceFileLine, "stop_lat,stop_lon", duplicateStop.getStop1Id());
```

Comment:
I think we want to pick which stop to use for the source line and affected entity based on which has a higher sourceFileLine. The second stop in the file is the duplicate.